google-map-marker.html

```diff
@@ -249,6 +249,14 @@
           },
 
           /**
+           * The marker's label.
+           */
+          label: {
+            type: String,
+            value: null,
+            observer: '_labelChanged'
+          },
```

Comment:
@abdonrd have to be like that! I missed that one doing the merge conflict, sorry about that.